src/components/DeleteAccount/helpers.spec.js

```diff
@@ -0,0 +1,72 @@
+import { createMockClient } from 'cozy-client'
+import { submitPassword, containerForTesting } from './helpers'
+
+const client = createMockClient({})
+const primaryAction = jest.fn()
+const setError = jest.fn()
+const setIsRequired = jest.fn()
+const setIsBusy = jest.fn()
+const t = jest.fn()
+
+const setup = async ({ currentPassphrase } = {}) =>
+  await submitPassword({
+    client,
+    t,
+    currentPassphrase,
+    primaryAction,
+    setError,
+    setIsRequired,
+    setIsBusy
+  })
+
+describe('submitPassword', () => {
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('should set required if no password value', async () => {
+    await setup({ currentPassphrase: '' })
+
+    expect(setError.mock.calls.length).toBe(1)
+    expect(setError).toHaveBeenCalledWith('')
+    expect(setIsRequired).toHaveBeenCalledWith(true)
+    expect(setIsBusy).not.toHaveBeenCalled()
+    expect(primaryAction).not.toHaveBeenCalled()
+  })
+
+  it('should set error for a wrong password ', async () => {
+    jest
+      .spyOn(containerForTesting, 'fetchPasswordValidation')
```

Comment:
nit: With `jest.mock('MODULE'` you should be able to avoid having a `containerForTesting` export
```js
import { submitPassword, containerForTesting } from './helpers'

jest.mock('./helpers', () => ({
  ...jest.requireActual('./helpers')
  fetchPasswordValidation: jest.fn()
}))

it('should set error for a wrong password ', async () => {
  fetchPasswordValidation.mockResolvedValue(...)
  //...
})
```